appcues/src/main/java/com/appcues/action/appcues/DelayAction.kt

```diff
@@ -0,0 +1,22 @@
+package com.appcues.action.appcues
+
+import com.appcues.action.ExperienceAction
+import com.appcues.data.model.AppcuesConfigMap
+import com.appcues.data.model.getConfigOrDefault
+import kotlinx.coroutines.delay
+
+internal class DelayAction(
+    override val config: AppcuesConfigMap,
+) : ExperienceAction {
+
+    companion object {
+
+        const val TYPE = "@appcues/delay"
+    }
+
+    private val duration = config.getConfigOrDefault("duration", 0)
```

Comment:
fwiw on iOS if there's no `duration` value, I'm failing the init of the delay action so that it gets skipped. Different than defaulting to 0 here, but the end result will be the same.

Reply:
yeah. thinking here there was not harm in just handling with default 0